general/releases/2.8/2.8.4.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Moodle 2.8.4
+tags: []
+sidebar_position: 4
+moodleVersion: 2.8.4
+---
+1. REDIRECT [Moodle 2.8.5 release notes](/general/releases/2.8/2.8.5)
```

Comment:
This file could be removed or improved because, in the old docs, it's redirected to 2.8.5